server/command.go

```diff
@@ -160,7 +160,12 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*mo
 				txt = "### Subscriptions in this channel\n"
 			}
 			for _, sub := range subs {
-				txt += fmt.Sprintf("* `%s` - %s\n", strings.Trim(sub.Repository, "/"), sub.Features)
+				subFlags := fmt.Sprintf("%s", sub.Flags)
```

Comment:
This is clever, but hard to read. Calling `String()` directly would make it more clear to me what the code does.